### ISSUE TITLE
Map is blank after isochrone calculation

### ISSUE THREAD

**Issue by astronaut4** [nilsnolde/ORStools#35]
Hi,

I am trying to make some isochrones and I tried using the OSM tools. When I click to make the isochrones, the whole map becomes blank white, even though the new layers show on the layers list. Even if I delete the isochrone layers, the map remains blank white. Could you help me with this?

Thank you,

Panos

**Comment by nilsnolde** [comment]
Hi Panos,

thanks for reporting!

I had this issue too. It happens when your map canvas is in a projected CRS. Try a geographic CRS (i.e. WGS84 or others) and it´s fine.

I added the functionality to zoom to the isochrone layer after calculation. I gotta read up on the function I used apparently. Thought it´d work the same as the context item 'Zoom to layer', but it doesn´t..

So, workaround for now: Right-click --> Zoom to layer on the isochrone. 

Will fix in next version.
